### DISCUSSION TITLE
add alias for npm run

### DISCUSSION THREAD

**Discussion by basickarl** [npm/rfcs#669] [RRFC]
writing `npm run <script>` doesn't seem like much but writing this day in and day out is a pain. Yarn already has fixed this by only having to type `yarn <script>`. 

I'd like to see an alias for `npm run`, something like `npmr`(short for npm run).

I can't be the only one wanting this?

**Comment by ruyadorno** [comment]
> I can't be the only one wanting this?

No you can't 😂 

> Yarn already has fixed this by only having to type yarn <script>.

Yarn hasn't fixed it, like many things in software it's a trade-off. I believe it's more accurate to say that Yarn made an opinionated choice that a portion of users prefer but [that choice also has its pitfalls](https://github.com/npm/rfcs/pull/279#issuecomment-823638195).

This has been discussed extensively in the past, last effort being the RFC from @orta which you can read the activity and conclusion over here: https://github.com/npm/rfcs/pull/332
